packages/flx/test/signing_test.dart

```diff
@@ -34,7 +34,7 @@ main() async {
 
   // Set up a key generator.
   CipherParameters cipher = CipherParameters.get();
-  await cipher.seedRandom();
+  cipher.seedRandom();
```

Comment:
This is not correct. The tests will sometimes run before we seed the RNG. Is it too slow? I can add a way to seed with some in-memory numbers - though that will get rid of the basic fuzz testing we had.